legal-api/tests/unit/resources/v2/test_business.py

```diff
@@ -338,6 +338,48 @@ def test_get_business_with_court_orders(session, client, jwt):
     assert rv.json['business']['identifier'] == identifier
     assert rv.json['business']['hasCourtOrders'] == True
 
+@pytest.mark.parametrize('identifier, legal_type, draft_type, nr_number', [
+    ('Tb31yQIuBw', 'COMP', 'ATMP', None),
```

Comment:
there is no legal_type called COMP. Did you mean BC (https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/business.py#L66)?